tests/test_audiofile.py

```diff
@@ -163,49 +171,63 @@ def test_empty_file(empty_file):
 
 
 @pytest.mark.parametrize(
-    'ext, expected_error',
-    [
-        ('bin', OSError),
-        ('mp3', OSError),
-        ('wav', RuntimeError),
-    ],
+    'ext',
+    ('bin', 'mp3', 'wav'),
 )
-def test_missing_file(ext, expected_error):
+def test_missing_file(tmpdir, ext):
     missing_file = f'missing_file.{ext}'
+    expected_error = RuntimeError
```

Comment:
Ok, so we always raise the error now. If I recall correctly you originally didn't want to apply this change as it breaks the API. But I also think it's better this way.

Reply:
Yes, but it does not break the API as we specified them in the docs. It just breaks with the behavior as it was during the last years, even if this behavior was wrong ;)

Reply:
I guess you mean "NOT specified them in the docs", right? Yes, noticed that, too. So it's rather a bug fix than a breaking change :)

Reply:
Yes, I meant "NOT specified them in the docs"